scripts/sanitycheck

```diff
@@ -1332,7 +1332,8 @@ class MakeGenerator:
 
         if len(ti.test.extra_configs) > 0 or options.coverage:
             args.append("OVERLAY_CONFIG=\"%s %s\"" %(overlays,
-                        os.path.join(ti.outdir, "overlay.conf")))
+                        os.path.join(ti.outdir,
+                                     "sanitycheck", "yaml_extra.conf")))
```

Comment:
instead of yaml, call it testcase_extra.conf, we should not naming the file with implementation details (before yaml we had ini, who known what will come next!)

Reply:
@marc-hb this is still pending, AFAICS

Reply:
Done, I was giving other people some time/a chance to comment as well.